client/starwhale/core/dataset/cli.py

```diff
@@ -69,7 +69,7 @@ def _build(
     append: bool,
     append_from: str,
     runtime: str,
-) -> None:
+) -> t.Union[URI, t.Any]:
```

Comment:
t.Any is the any type.

`t.Union[URI, t.Any] ` = `t.Any`

Reply:
done